docs-gen/content/rule_set/data_entry/_index.md

```diff
@@ -11,7 +11,7 @@ In order to help application developers, who are using the specification, it mak
 The difference between a signal and an attribute is that the signal has
 a publisher (or producer) that continuously updates the signal value while an
 attribute has a set value, defined in the specification, that never changes.
-As summary, besides [```branch```](/rule_set/branches) type can be:
+As summary, besides [```branch```](/vehicle_signal_specification/rule_set/branches) type can be:
```

Comment:
[```branch```] is linked here (correctly), however in the same paragraph frither up sensor/actuator and stream (is this ever used?) are not.

Reply:
I was deliberately constraining myself to existing links that were broken in this way, but yes I can push some additional links. What is the correct path for stream?

Reply:
Added links in PR update.